src/client/components/VoyageTeaser.js

```diff
@@ -1,6 +1,37 @@
-import React, { Component } from "react";
+import React from "react";
 import PropTypes from "prop-types";
 
+//import material-ui
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+
+//CSS Colors: #F2F2F2,#D9D9D9,#595959,#262626,#0D0D0D
+const voyage_teaser = makeStyles({
```

Comment:
You can change it to voyageTeaserStyle or something like that

Reply:
@AbodHassam First I need to be sure Zaki is accepting this style then I can change the name as well.

Reply:
Agree. It should be camelCased.

I wanted to do something like this to simplify things:

```
const classes = makeStyles({ ...styles...})
```

But I did a quick test and it was breaking and I could not figure out why. Can you try if you can get it to work? Would be nice to kill that intermediary variable assignment (`classes = voyage_teaser()`).